### PR TITLE
Always decode objectPath

### PR DESCRIPTION
**Pull request by rogerwangcs**
I'm seeing an issue on the verily devel environment where the `objectPath` path parameter in the GetFileInBucket endpoint is not properly uri decoded when it is passed into the controller. It seems like the generated openapi rest controller doesn't correctly decode path params.

As an example, passing in `"folder%2Ffilename.txt"` as the `objectPath` results in a failed cloud storage call and the following axon server error: "Cannot read object: folder%2Ffilename.txt.

I'm unable to reproduce this in my local environment, but this change decodes any encoded string before making the cloud storage call.

### CODE REVIEW

**Comment by yuhuyoyo** [comment]
can you give more context? e.g. a ticket, or the before and after

File: service/src/main/java/bio/terra/axonserver/utils/PathValidationUtils.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import bio.terra.common.exception.BadRequestException;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+
+public class PathValidationUtils {
+  public static String decodePath(String path) throws BadRequestException {
+    try {
+      return URLDecoder.decode(path, StandardCharsets.UTF_8.toString());
+    } catch (Exception e) {
+      System.out.println("Exception: " + e.getMessage());
```
Comment (yuhuyoyo) [inline service/src/main/java/bio/terra/axonserver/utils/PathValidationUtils.java]: please use logger, isntead of system.out

Comment (rogerwangcs) [reply service/src/main/java/bio/terra/axonserver/utils/PathValidationUtils.java]: > can you give more context? e.g. a ticket, or the before and after

updated description

Comment (yuhuyoyo) [reply service/src/main/java/bio/terra/axonserver/utils/PathValidationUtils.java]: let's just put this method in CloudStorageUtils. I don't think it's worth it to split out for one usage. especially this is one line of code.

I usually split out utils when there's 3+ methods in the same category. thanks

**Review by yuhuyoyo — APPROVED** [review]

File: service/src/main/java/bio/terra/axonserver/utils/PathValidationUtils.java (hunk 1 above)
Comment (rogerwangcs) [reply service/src/main/java/bio/terra/axonserver/utils/PathValidationUtils.java]: Sg

**Review by zloery — APPROVED** [review]

File: service/src/main/java/bio/terra/axonserver/utils/CloudStorageUtils.java [hunk 2]
```diff
@@ -52,6 +55,13 @@ public static GoogleCredentials getGoogleCredentialsFromToken(String token) {
   public static byte[] getBucketObject(
       GoogleCredentials googleCredentials, String bucketName, String objectName) {
 
+    // encoded slashes in object path and decode them
```
Comment (zloery) [inline service/src/main/java/bio/terra/axonserver/utils/CloudStorageUtils.java]: nit: I think this comment is missing a word at the start

**Pull request merged**